src/Tools/DynamoFeatureFlags/DynamoFeatureFlags.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <ImportGroup Label="PropertySheets">
+    <Import Project="$(SolutionDir)Config/CS_SDK.props" />
+  </ImportGroup>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <OutputPath>$(OutputPath)\DynamoFeatureFlags\</OutputPath>
```

Comment:
Remind me, the output will not be part of DynamoCore right?

Reply:
It will be placed in `Bin\DynamoFeatureFlags` subfolder so it will be part of DynamoCore release zip and artifactory build - but it won't pollute the root bin folder.